repo/grpc_repository_client.go

```diff
@@ -586,10 +591,17 @@ func (r *grpcRepositoryClient) UpdateDescription(d string) {
 }
 
 func (r *grpcRepositoryClient) Close(ctx context.Context) error {
+	if r.omgr == nil {
```

Comment:
is this safe with concurrency?

Reply:
Close() is meant to be called without concurrency